balancer/balancer.go

```diff
@@ -72,8 +72,22 @@ func unregisterForTesting(name string) {
 	delete(m, name)
 }
 
+// connectedAddress returns the connected address for a SubConnState.
+// The second return value is set to to false if the state is not READY, and the
+// first return value is meaningless in this case.
```

Comment:
Why is anyone even calling this if the state isn't ready?  Can we just say it's only valid if READY to simplify things a bit?

Reply:
Done.